### ISSUE TITLE
Labels leaking in training data

### ISSUE THREAD

**Issue by hamzahass** [philschmid/deep-learning-pytorch-huggingface#73]
Hello,

Thanks for providing some fine-tuning code for VLMs in your article "How to Fine-Tune Multimodal Models or VLMs with Hugging Face TRL". Maybe I misunderstood something, but it seems that your training data inputs contain your labels (which in your case are the products' descriptions). Is this expected ?? I am working on fine-tuning Molmo on object detection, and this doesn't make sense to me.
